public/controllers/aeon_requests_controller.rb

```diff
@@ -175,14 +175,14 @@ def aeon_request_hash(record)
     title = ancestry_titles_for(record).first
 
     site        = determine_site_from_repo_code(@repository["repo_code"])
-    location    = locations_data_for(record).map{ |l| "#{l[:area]}, #{l[:sub_area]}" }.join("; ")
+    location    = locations_data_for(record).map{ |l| "#{l[:area]}" }.join("; ")
     item_volume = locations_data_for(record).map{ |l| "#{l[:sub_area]}" }.join("; ")
     callnum     = callnum_for(record)
 
     {
       title:       title,
       site:        site,
-      location:    location,
+      sub_location:    location,
```

Comment:
I think this requires an update to the README under "How it maps". Might "location" sometimes be the correct choice for some institutions?